test/dotnet-new3.UnitTests/Approvals/DotnetNewHelp.CanShowHelp.approved.txt

```diff
@@ -1,25 +1,16 @@
-﻿Usage: new3 [options]
+﻿Description:
+  Template Instantiation Commands for .NET CLI.
 
-Options:
-  -h, --help                     Displays help for this command.
-  -l, --list <PARTIAL_NAME>      Lists templates containing the specified template name. If no name is specified, lists all templates.
-  -n, --name                     The name for the output being created. If no name is specified, the name of the output directory is used.
-  -o, --output                   Location to place the generated output.
-  -i, --install                  Installs a source or a template package.
-  -u, --uninstall                Uninstalls a source or a template package.
-  --interactive                  Allows the internal dotnet restore command to stop and wait for user input or action (for example to complete authentication).
-  --add-source, --nuget-source   Specifies a NuGet source to use during install.
-  --type                         Filters templates based on available types. Predefined values are "project" and "item".
-  --dry-run                      Displays a summary of what would happen if the given command line were run if it would result in a template creation.
-  --force                        Forces content to be generated even if it would change existing files.
-  -lang, --language              Filters templates based on language and specifies the language of the template to create.
-  --update-check                 Check the currently installed template packages for updates.
-  --update-apply                 Check the currently installed template packages for update, and install the updates.
-  --search <PARTIAL_NAME>        Searches for the templates on NuGet.org.
-  --author <AUTHOR>              Filters the templates based on the template author. Applicable only with --search or --list | -l option.
-  --package <PACKAGE>            Filters the templates based on NuGet package ID. Applies to --search.
-  --columns <COLUMNS_LIST>       Comma separated list of columns to display in --list and --search output.
-                                 The supported columns are: language, tags, author, type.
-  --columns-all                  Display all columns in --list and --search output.
-  --tag <TAG>                    Filters the templates based on the tag. Applies to --search and --list.
-  --no-update-check              Disables checking for the template package updates when instantiating a template.
\ No newline at end of file
+Usage:
```

Comment:
This help output is laid out a little oddly to my eye.

The usage suggests that a command could come after the template short name and template args, `dotnet new3 webapi --langVersion latest update`, but that is not valid, is it?

In cases like these, where a root command has multiple branching usages, I've seen some CLI tools offer examples of the two modes separately:

```
Usage: 
new3 [<template-short-name> [<template-args>...]]
new3 [command]
```

Reply:
yes, both template and subcommand is not valid. I will do custom usage here as above.

Reply:
fixed

Reply:
@baronfel do you prefer to keep "options" or remove them below?

Reply:
Options works great here 👍